src/ucp/rndv/rndv_ppln.c

```diff
@@ -179,6 +179,13 @@ void ucp_proto_rndv_ppln_send_frag_complete(ucp_request_t *freq, int send_ack)
                                       "ppln_send");
 }
 
+void ucp_proto_rndv_ppln_recv_frag_clean(ucp_request_t *freq)
+{
+    ucp_proto_rndv_ppln_frag_complete(
```

Comment:
this function can send ACK to remote peer, is this ok?

Reply:
it can't because parameter `send_ack=0`